docs/run.js

```diff
@@ -31,30 +28,37 @@ const scalingFactor = 100;
 let resolvedScoreAsset;
 
 /**
- * Sets {@link scorePromise} to the {@link ee.FeatureCollection} with
- * id {@link getScoreAssetPath}, or, if that does not exist, with id
- * {@link getBackupScoreAssetPath}.  Sets {@link resolvedScoreAsset} to the id
- * of the found collection.
+ * Seeds {@link Promise} cache with `eeAsset`, and returns {@link Promise} whose
+ * value is `eeAsset`. Caller can then wait on returned {@link Promise} to see
+ * if `eeAsset` is valid asset.
+ * @param {string} eeAsset
+ * @return {Promise<string>} {@link Promise} with `eeAsset` once {@link
```

Comment:
super nit: this {@link Promise} isn't highlighting correctly in github - is it in intellij? if not there either maybe just take out

Reply:
Highlights same as other places.